app/models/miq_worker.rb

```diff
@@ -267,6 +267,8 @@ def self.start_worker(*params)
     w
   end
 
+  cache_with_timeout(:my_worker) { server_scope.find_by(:pid => Process.pid) }
```

Comment:
@kbrock what happens with subclasses that call `my_worker`?  `MiqGenericWorker.my_worker` as an example.  Is it sharing a class variable or is it not shared?